lisp/mastodon-tl.el

```diff
@@ -130,6 +131,7 @@ Return value from boosted content if available."
 (defun mastodon-tl--byline (toot)
   "Generate byline for TOOT."
   (let ((id (cdr (assoc 'id toot)))
+        (timestamp (cdr (assoc 'created_at toot)))
```

Comment:
When the web client displays the timestamp for a boosted toot, does it show the time of the boost or the time of initial creation?

If it's the time of the boost, this will work just fine. If it's the time of initial creation, you'll want to use `mastodon-tl--field` so it grabs the boosted field.